check_py_coverage.sh

```diff
@@ -87,10 +87,18 @@ coverage run $EXTRA $FRAMEWORK_DIR/Driver.py test_maap5_code_interface_adaptive_
 coverage run $EXTRA $FRAMEWORK_DIR/Driver.py test_maap5_code_interface_det_multibranch.xml interfaceCheck
 coverage run $EXTRA $FRAMEWORK_DIR/Driver.py test_relap5_code_inss.xml interfaceCheck
 # code interface tests END
+
+Xvfb :8888 &
```

Comment:
only, besides just echoing if it is found or not, do the code to start it and change the DISPLAY